hack/lint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+for d in $(find . -type d -a \( -iwholename './pkg*' -o -iwholename './cmd*' \)); do
+	echo ${d}
```

Comment:
when you run make verify it outputs the context of "checking lint" which is a little slow so I figured let it show what directories lint is checking.  Other than checking lint what additional context are you thinking?